### ISSUE TITLE
BAZARR an exception occurs during the synchronization process for this subtitles

### ISSUE THREAD

**Issue by noonien** [morpheus65535/bazarr#1392]
**Describe the bug**
Commit 4a0932b5d3052867f7f92984300d2ab4ec54fb0d seems to have introduced a bug in ffsubsync, I believe the version of ffsubsync that was vendored is not compatible with the version with auditok that was vendored.

ffsubsync creates an auditok.AudioEnergyValidator with a missing parameter, `channels` here:
https://github.com/morpheus65535/bazarr/blob/master/libs/ffsubsync/speech_transformers.py#L79
The `channels` parameter is defined here:
https://github.com/morpheus65535/bazarr/blob/master/libs/auditok/util.py#L297

**To Reproduce**
1. Install latest version of Bazarr
2. Try to sync a subtitle
4. Exception occurs.

**Expected behavior**
ffsubsync should be called successfully.

**Screenshots**
Not applicable.

**Software (please complete the following information):**
 - Bazarr: v0.9.5-beta.3

**Additional context**
```
Traceback (most recent call last):  
File "/nix/store/y7lwl3v8s614x2l7s22n8w95bwspplwr-bazarr-0.9.5-beta.3/src/bazarr/subsyncer.py", line 56, in sync
    result = run(self.args)  
File "/nix/store/y7lwl3v8s614x2l7s22n8w95bwspplwr-bazarr-0.9.5-beta.3/src/bazarr/../libs/ffsubsync/ffsubsync.py", line 327, in run
    reference_pipe.fit(args.reference)  
File "/nix/store/y7lwl3v8s614x2l7s22n8w95bwspplwr-bazarr-0.9.5-beta.3/src/bazarr/../libs/ffsubsync/sklearn_shim.py", line 214, in fit
    self._final_estimator.fit(Xt, y, **fit_params)  
File "/nix/store/y7lwl3v8s614x2l7s22n8w95bwspplwr-bazarr-0.9.5-beta.3/src/bazarr/../libs/ffsubsync/speech_transformers.py", line 229, in fit
    detector = _make_auditok_detector(self.sample_rate, self.frame_rate, self._non_speech_label)  
File "/nix/store/y7lwl3v8s614x2l7s22n8w95bwspplwr-bazarr-0.9.5-beta.3/src/bazarr/../libs/ffsubsync/speech_transformers.py", line 78, in _make_auditok_detector
    validator = AudioEnergyValidator(TypeError: __init__() missing 1 required positional argument: 'channels'
```

**Comment by morpheus65535** [comment]
Can't reproduce using 0.9.5-beta6 and webrtcvad-wheels 2.0.10.post2

**Comment by noonien** [comment]
Running `0.9.5-beta.7` and hitting the same issue.
The way NixOS works is that I do not have a global `pip` environment, here is the package for `bazarr`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/bazarr/default.nix I've changed only the version so that I'm using 0.9.5-beta.7.

If you are note familiar with NixOS, the package does not have any external dependencies apart from `python3, unrar, ffmpeg`, this means also no external python libraries.

However, bazarr still works because of this function: https://github.com/morpheus65535/bazarr/blob/development/bazarr/libs.py#L19-L20 which means that the dependencies are vendered and loaded from here: https://github.com/morpheus65535/bazarr/tree/development/libs and, I cannot tell what version of webrtcvad-wheels is there, because https://github.com/morpheus65535/bazarr/blob/development/libs/version.txt doesn't specify a version for it, but, it does specify versions for auditok and ffsubsync.

**Comment by halali** [comment]
Ok this package is probably outdated, because dependency you missing have to be installed from requirements.txt same like lxml

**Comment by noonien** [comment]
As I said, I installed no dependencies, they are all in this git repository. I don't have pip installed globally on my system, nor a global python environment.

**Comment by morpheus65535** [comment]
You have to install requirements.txt. If your OS doesn't support it, you'll run into trouble again in 0.9.7 as we will introduce other module that can't be vendored. Bazarr will try to install them on every start and you'll get into an infinite loop. Looking at documentation, you can easily do this by running Bazarr in a venv: https://nixos.wiki/wiki/Python#Python_virtual_environment

**Comment by noonien** [comment]
Fixed in 30ef713fa260dc9d9d3120359cd052808606badb
